dll/win32/kernel32/kernel32.spec

```diff
@@ -1182,7 +1182,7 @@
 @ stdcall -ret64 VerSetConditionMask(long long long long) ntdll.VerSetConditionMask
 @ stdcall VerifyConsoleIoHandle(long) kernel32_win7.VerifyConsoleIoHandle
 @ stub -version=0x600+ VerifyScripts
-@ stdcall VerifyVersionInfoA(long long double) kernel32_win7.VerifyVersionInfoA
+@ stdcall VerifyVersionInfoA(long long double)
```

Comment:
You started mentioning it in mattermost, but since we're here in github....
Could you explain why (which aspects....) the fact nfsd is NT6 but compiled as ANSI caused problems here?

Reply:
Because KernelBase doesn't export a few of the needed ANSI functions is all it comes down to, i.e see geoff.
https://www.geoffchappell.com/studies/windows/win32/kernelbase/api/index.htm

Reply:
But then, linking nfsd.exe to either just kernel32, or to both kernel32 and kernelbase, wouldn't solve the problem as well?

Reply:
It's incorrect to link to both kernelbase and kernel32.

Reply:
So let's say, linking only to kernel32 ?

Reply:
That's how it works in this PR, it is only linking to kernel32.

I mean unless you're ready to drop windows 2003 target right now.